.github/workflows/release.yml

```diff
@@ -48,19 +61,22 @@ jobs:
         env :
           GITHUB_TOKEN : ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Get version
-        id: get_version
-        if: ${{ success() }}
-        run: echo "version=${GITHUB_REF/refs\/tags\//}" >> $GITHUB_ENV
-
       - name: Bump Homebrew Formula
         if: ${{ success() }}
         uses: mislav/bump-homebrew-formula-action@v2
         env:
           COMMITTER_TOKEN: ${{ secrets.HOMEBREW_TOKEN }}
         with:
           formula-name: ktlint
-          download-url: https://github.com/pinterest/ktlint/releases/download/${{ env.version }}/ktlint
+          download-url: https://github.com/pinterest/ktlint/releases/download/${{ env.version }}/ktlint-${{ env.version }}.zip
+
+      - name: Release to sdkman
+        if: ${{ success() }}
+        env:
+          SDKMAN_KEY: ${{ secrets.SDKMAN_KEY }}
+          SDKMAN_TOKEN: ${{ secrets.SDKMAN_TOKEN }}
```

Comment:
@paul-dingemans these need to be added to the repro action secrets after receiving them from https://github.com/sdkman/sdkman-cli/wiki/Vendor-onboarding-process#request-api-credentials

Reply:
got it. just sent the credentials request.